awesome_cart/awesome_cart/doctype/awc_transaction/awc_transaction.py

```diff
@@ -21,13 +21,30 @@
 	"Debug": 3
 }
 
+def call_hook(hook_name, **kwargs):
+	hooks = frappe.get_hooks(hook_name) or []
+	if hooks:
+		for hook in hooks:
+			# don't allow hooks to break processing
+			try:
+				frappe.call(hook, **kwargs)
+			except Exception as hook_ex:
+				log("Error calling hook method: {}->{}".format(hook_name, hook))
```

Comment:
Man I'm not sure about this silently failing. Atleast Sentry should pick this up somehow.